src/_includes/macros/eventinfo.njk

```diff
@@ -21,16 +21,20 @@
     <h2>{{ event.description }}</h2>
 </header>
 {% if event.stream %}
-    {% if event.date | occured %}
+    {% if event.happened == true %}
     <p>It was ✨ Broadcast Live ✨ on the ✨ Information Superhighway ✨</p>
+    {% else %}
+        {% if event.date | occurred %}
+            <p>Currently live 📣</p>
+        {% endif %}
     {% endif %}
     <div class="videoWrapper">
     <iframe width="1120" height="630" src="{{ event.stream }}" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
     </div>
```

Comment:
These three have been replaced by https://github.com/jsoxford/jsnotoxfordsite/blob/main/src/_includes/macros/eventinfo.njk#L28

Weird it doesn't pull this is in as a change

Reply:
I think my branch is just behind `main`, I'll rebase it when I get a moment 
![image](https://user-images.githubusercontent.com/50949/107951831-d8e82b80-6f90-11eb-8c25-640e83b2c0e4.png)

Reply:
That should now be resolved 
![image](https://user-images.githubusercontent.com/50949/107952680-fa95e280-6f91-11eb-8578-9d7b8a5c2638.png)